gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageClientImpl.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2022 Google LLC
```

Comment:
nit: 2023? :)

Reply:
This file was renamed from [GcsJavaClientImpl.java](https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/fabf01d601537f9f663ff700e457809edbd172fe/gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GcsJavaClientImpl.java) that was missing 2022 license (added in https://github.com/GoogleCloudDataproc/hadoop-connectors/pull/939).